build-release-tools/docker_build.sh

```diff
@@ -34,7 +34,7 @@ tagCalculate() {
     else
```

Comment:
for this file ```docker_build.sh```
I find no changes of this file.
So I suggest you to wget a new file of master branch to overwrite this file.